src/app/app.module.ts

```diff
@@ -30,35 +26,22 @@ import { FlowerTestComponent } from './flower-test/flower-test.component';
 import { HomeComponent } from './home/home.component';
 import { TopMenuBarComponent } from './top-menu-bar/top-menu-bar.component';
 import { FirebaseTestComponent } from './firebase-test/firebase-test.component';
-
-import {MdcButtonModule} from '@angular-mdc/web/button';
-import {MdcFabModule} from '@angular-mdc/web/fab';
-import {MdcIconModule} from '@angular-mdc/web/icon';
-import {MdcTopAppBarModule} from '@angular-mdc/web/top-app-bar';
-import {MdcLinearProgressModule} from '@angular-mdc/web/linear-progress';
-import {MdcElevationModule} from '@angular-mdc/web/elevation';
-import {MdcSliderModule} from '@angular-mdc/web/slider';
-import {MdcIconButtonModule} from '@angular-mdc/web/icon-button';
-import {MdcSnackbarModule} from '@angular-mdc/web/snackbar';
 import { JsonDataTestComponent } from './json-data-test/json-data-test.component';
 
+import { MatProgressBarModule } from '@angular/material/progress-bar';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {MatButtonModule} from '@angular/material/button';
+import {MatIconModule} from '@angular/material/icon';
```

Comment:
The spacing here should probably be consistent with other imports, which mostly have spaces between words and brackets.